src/native/libs/System.Native/pal_networkchange.c

```diff
@@ -179,8 +179,7 @@ Error SystemNative_ReadEvents(intptr_t sock, NetworkChangeEvent onNetworkChange)
 {
     (void)sock;
     (void)onNetworkChange;
-    // unreachable
     abort();
-    return Error_EINVAL;
+    __builtin_unreachable();
```

Comment:
```suggestion
    return Error_ENOTSUP;
```
We can just return an error code like we do in number of existing places. It is the most reliable way to stop fighting with compiler about unreachable warnings.